src/index.ts

```diff
@@ -0,0 +1,108 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import { MapboxDraw } from "@mapbox/mapbox-gl-draw/dist/mapbox-gl-draw";
+
+export class MapBoxPropTypes {
```

Comment:
Since we're using TS, there's no need for proptypes anymore. `interface Props {...}`, and use TS types instead of PropTypes types.

Reply:
Quick explanation:

For types checking for props and context in React components, there are two ways to do it:
- either via propTypes (the solution initially proposed by Facebook, works with vanilla JS)
- or via TS types.

It's useless to do both, I was suggesting to switch from PropTypes to TS.